node-services/supervisor/src/domain/application/connectionChecker.ts

```diff
@@ -55,7 +55,8 @@ export default class ConnectionChecker {
     }
 
     public setConsiderConnectedIfUserInGroups(considerConnectedIfUserInGroups: any): this {
-        this.groupsList = considerConnectedIfUserInGroups;
+        if (considerConnectedIfUserInGroups == null) this.groupsList = [];
+        else this.groupsList = considerConnectedIfUserInGroups;
```

Comment:
Can be written as
this.groupsList = (considerConnectedIfUserInGroups == null) ? [] : considerConnectedIfUserInGroups